app/templates/components/map-form.hbs

```diff
@@ -5,16 +5,16 @@
 
     <hr/>
 
-    <input type="radio" name="paper-orientation" value="portrait" id="paper-orientation--portrait" onclick={{action 'reorientPaper' 'portrait'}}/>
+    <input type="radio" name="paper-orientation" class=portrait-radio-button value="portrait" id="paper-orientation--portrait" onclick={{action 'reorientPaper' 'portrait'}}/>
```

Comment:
We should have quotes around these classnames... sorry, template linting isn't enabled for this